.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,103 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+---
+<!--
+Thank you for reporting an issue with OpenRewrite!
+We appreciate you taking the time to help us improve.
+Please fill out the template below to help us understand and reproduce the issue.
+Feel free to delete any sections that don't apply to your issue.
+-->
+
+## What version of OpenRewrite are you using?
+<!--
+Whenever possible please try to replicate your issue with the latest versions of OpenRewrite.
+The latest versions of some common OpenRewrite projects are listed here.
+- https://github.com/openrewrite/rewrite-maven-plugin/releases
```

Comment:
We could consider adding a link to our docs which has the latest versions _usually_: https://docs.openrewrite.org/reference/latest-versions-of-every-openrewrite-module